packages/fiori-freestyle-writer/test/__snapshots__/index.test.ts.snap

```diff
@@ -898,7 +898,7 @@ title=My Test App",
   \\"sap.ui5\\": {
     \\"flexEnabled\\": true,
     \\"dependencies\\": {
-      \\"minUI5Version\\": \\"1.60.0\\",
+      \\"minUI5Version\\": \\"\\",
```

Comment:
`minUI5Version` with no value looks incorrect

Reply:
Why? If no minUI5Version is provided then the Fiori tools will use the latest version which would be the expected behavior if no min version or version is provided.

Reply:
This may be a mandatory property and should contain a value. Please see: https://ui5.sap.com/#/topic/be0cf40f61184b358b5faedaec98b2da

Reply:
> Why? If no minUI5Version is provided then the Fiori tools will use the latest version which would be the expected behavior if no min version or version is provided.

Preview technically works.  But `minUI5Version` is mandatory as Iain mentioned and I think is checked during building or deploying the app. So better to keep a value here IMO.

Reply:
@petermuessig we need your guidance here. @IainSAP is correct, the doc says that the `minUI5Version` is mandatory, so leaving it empty like I planned to seems not correct, however, I took your recent change to `ui5-language-assistant` (https://github.com/SAP/ui5-language-assistant/pull/472/) as guidance, where you also use the absence of the `minUI5Version` as an indicator to use the latest version.

Reply:
Ok, I have removed the fix and we keep generating `minUI5Version = 1.60` if nothing is provided. I am still fully convinced it is not at all what a caller would expect but changing it is a bit more than just leaving it empty. This really needs attention soon but for now I have just enhanced the example generator to show to consumers how to fetch the latest version during the prompting phase.